product/module.go

```diff
@@ -75,7 +75,8 @@ commerce: {
 		slugAttributeCode: string | *"urlSlug"
 		fakeservice: {
 			enabled: bool | *false
-			currency: *"€" | !=""  
+			currency: *"€" | !=""
+			jsonTestDataFolder: *"testdata/products" | !=""
```

Comment:
Here, I must specify a `jsonTestDataFolder` even if the feature is disabled.
I suggest to use an if statement here or make it an optional value, which would match the injection.